### PR TITLE
feat: add doc for logs api

### PR DESCRIPTION
**Pull request by nityanandagohain**
Docs for logs api

### CODE REVIEW

File: docs/userguide/logs_api.md [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+
+### Top-level model 
+
+The top-level model looks like this and is broken down into further tables
+
+|  NAME  | DESCRIPTION  |
+|---|---|
+|start| Epoch timestamp start in ms/ns |
+|end | Epoch timestamp end in ms/ns |
+|step | Step size for the query in seconds |
```
Comment (srikanthccv) [inline docs/userguide/logs_api.md]: It would be good to call this out as an aggregation interval. Step might not be oblivious some people.

**Review by srikanthccv — APPROVED** [review]

File: docs/userguide/logs_api.md [hunk 2]
```diff
@@ -0,0 +1,475 @@
+---
+title: Logs API Documentation
+id: logs_api
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+In this doc, we will go through the logs API through which you can fetch logs and run different kinds of aggregation.
```
Comment (Calm-Rock) [inline docs/userguide/logs_api.md]: It should be clearly mentioned 
- What is Logs API
- Why does someone need this, i.e, when would a user want to use this

File: docs/userguide/logs_api.md [hunk 3]
```diff
@@ -0,0 +1,475 @@
+---
+title: Logs API Documentation
+id: logs_api
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
```
Comment (Calm-Rock) [inline docs/userguide/logs_api.md]: ## Overview

Make this the overview section

File: docs/userguide/logs_api.md [hunk 4]
```diff
@@ -0,0 +1,475 @@
...
+In this doc, we will go through the logs API through which you can fetch logs and run different kinds of aggregation.
+
```
Comment (Calm-Rock) [inline docs/userguide/logs_api.md]: #### Endpoint

This should be clearly mentioned as an endpoint

File: docs/userguide/logs_api.md [hunk 5]
```diff
@@ -0,0 +1,475 @@
...
+```
+POST /api/v3/query_range
```
Comment (Calm-Rock) [inline docs/userguide/logs_api.md]:
```suggestion
`POST` `/api/v3/query_range`
```
Remove this from a code block and put it in inline code, ie, `POST`  `/api/v3/query_range`

File: docs/userguide/logs_api.md (hunk 4 above)
Comment (Calm-Rock) [inline docs/userguide/logs_api.md]: If we have any pre-requisite, mention them here.
 For example if we need to have an API key , please mention it here and also explain how user can obtain it.

File: docs/userguide/logs_api.md [hunk 6]
```diff
@@ -0,0 +1,475 @@
...
+```
```
Comment (Calm-Rock) [inline docs/userguide/logs_api.md]: Explain in brief what this end-point does.

File: docs/userguide/logs_api.md (hunk 2 above)
Comment (nityanandagohain) [reply docs/userguide/logs_api.md]: have added

File: docs/userguide/logs_api.md (hunk 6 above)
Comment (nityanandagohain) [reply docs/userguide/logs_api.md]: already added in overview section

**Comment by nityanandagohain** [comment]
have made the changes @Calm-Rock

**Pull request merged**